server/repository/src/db_diesel/pack_variant_row.rs

```diff
@@ -18,9 +19,21 @@ table! {
 joinable!(pack_variant -> item (item_id));
 
 #[derive(
-    Clone, Queryable, Insertable, AsChangeset, Debug, PartialEq, Default, Eq, Ord, PartialOrd,
+    Clone,
+    Queryable,
+    Insertable,
+    AsChangeset,
+    Debug,
+    PartialEq,
+    Default,
+    Eq,
+    Ord,
+    PartialOrd,
+    Serialize,
+    Deserialize,
 )]
 #[table_name = "pack_variant"]
+#[serde(rename_all = "camelCase")]
 pub struct PackVariantRow {
```

Comment:
Pack variant is translated purely with serde (well apart from SyncTranlsation implementation for dependencies and to indicate which sites should push it